tmeister/static/src/toggles/metrics/metrics.hooks.js

```diff
@@ -20,8 +20,9 @@ export function useMetricsForToggle (toggle) {
         if (response.ok) {
           return response.json()
             .then(property('metrics'))
-            .then(v => {
-              setMetrics(v)
+            .then(metrics => {
+              const grouped = groupBy(metrics, (item) => item.environment)
```

Comment:
I forgot to group them, I did this with the mock response but I forgot to do it here.